libbpf-tools/klockstat.bpf.c

```diff
@@ -147,6 +148,10 @@ static void account(struct lockholder_info *li)
 {
 	struct lock_stat *ls;
 	u64 delta;
+	u32 key = li->stack_id;
+
+	if (per_thread)
+		key = li->task_id;
 
 	/*
```

Comment:
maybe move this comment to where you assign key = li->stack_id?  since we aren't racy when dealing with task_id.

Reply:
Probably.. but I prefer not as it mentions the stack-id in the beginning. Or I can append something like this?

"But it should be ok for per-thread mode since it's not racy anymore."

Reply:
SGTM